models/safe/zkevm/safe_zkevm_matic_transfers.sql

```diff
@@ -0,0 +1,78 @@
+{{
+    config(
+        materialized='incremental',
+
+        alias = 'matic_transfers',
+        partition_by = ['block_month'],
+        unique_key = ['block_date', 'address', 'tx_hash', 'trace_address'],
+        on_schema_change='fail',
+        file_format ='delta',
+        incremental_strategy='merge',
+        post_hook='{{ expose_spells(\'["zkevm"]\',
+                                    "project",
+                                    "safe",
+                                    \'["danielpartida"]\') }}'
```

Comment:
There may be a better form for the `post_hook` now:
```suggestion
        post_hook = '{{ expose_spells(
                        blockchains = \'["zkevm"]\',
                        spell_type = "project",
                        spell_name = "safe_matic_transfers",
                        contributors = \'["danielpartida"]\') }}'
```